### DISCUSSION TITLE
CodeSanboxes down?

### DISCUSSION THREAD

**Discussion by jamezening** [TanStack/query#3733] [Q&A] [answered]
🚨 SOS 🚨
seems all the example CodeSandboxes are down?

![image](https://user-images.githubusercontent.com/6829451/175021953-b3246529-1feb-4f47-9754-3d89017848f3.png)

https://react-query.tanstack.com/examples/simple

**Comment by TkDodo** [comment, marked as answer]
they are pointing to `master`, but we renamed master to `main` recently. We're currently restructuring, so please bear with us.

you can click `open in sandbox` and change `master` to `main` in the url.
